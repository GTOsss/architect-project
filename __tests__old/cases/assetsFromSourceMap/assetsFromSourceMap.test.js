const appRoot = process.cwd();
const { resolve } = require('path');
const {
  returnOutputPathsWithContent,
  promisifyCliCommand,
  SnapshotSerializer,
} = require('../../utils/utilsForArcStart');
const cleanOutPutBeforeTest = require('../../utils/cleanOutPutBeforeTest');
const { pathForCommand, casesOutputPath } = require('../casesConfigPath');

const currentCommand = pathForCommand.assetsFromSourceMap;

describe('assetsFromSourceMap', () => {
  cleanOutPutBeforeTest(casesOutputPath.assetsFromSourceMap);

  test('test call function from script', async () => {
    await promisifyCliCommand(`arc s -c ${currentCommand}`);
    const currentPathToOutput = resolve(appRoot, currentCommand, 'output');

    SnapshotSerializer();

    expect(returnOutputPathsWithContent(currentPathToOutput)).toMatchSnapshot();
  });
});
