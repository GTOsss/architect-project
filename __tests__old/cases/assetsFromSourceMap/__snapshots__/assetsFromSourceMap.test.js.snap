// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`assetsFromSourceMap test call function from script 1`] = `
Object {
  "src": Array [
    Object {
      "content": "test",
      "path": "*/assetsFromSourceMap/output/src/services/icons/svg/a.svg",
    },
    Object {
      "content": "test",
      "path": "*/assetsFromSourceMap/output/src/services/icons/svg/arrow-right.svg",
    },
    Object {
      "content": "test",
      "path": "*/assetsFromSourceMap/output/src/services/icons/svg/blue-posts.svg",
    },
    Object {
      "content": "const t = "t"
",
      "path": "*/assetsFromSourceMap/output/src/services/name/index.ts",
    },
  ],
}
`;
